packages/vm/src/runTx.ts

```diff
@@ -395,22 +395,19 @@ async function _runTx(this: VM, opts: RunTxOpts): Promise<RunTxResult> {
       `Running tx=0x${
         tx.isSigned() ? tx.hash().toString('hex') : 'unsigned'
       } with caller=${caller} gasLimit=${gasLimit} to=${
-        to ? to.toString() : ''
+        to?.toString() ?? 'none'
       } value=${value} data=0x${short(data)}`
     )
   }
 
   const results = (await evm.executeMessage(message)) as RunTxResult
   if (this.DEBUG) {
+    const { gasUsed, exceptionError, returnValue, gasRefund } = results.execResult
     debug('-'.repeat(100))
     debug(
-      `Received tx results gasUsed=${results.gasUsed} execResult: [ gasUsed=${
-        results.gasUsed
-      } exceptionError=${
-        results.execResult.exceptionError ? results.execResult.exceptionError.error : ''
-      } returnValue=${short(results.execResult.returnValue)} gasRefund=${
-        results.execResult.gasRefund
-      } ]`
+      `Received tx execResult: [ gasUsed=${gasUsed} exceptionError=${
+        exceptionError ? `'${exceptionError.error}'` : 'none'
```

Comment:
What are the `'...'` for in the exceptionError?  Is it just to clearly demarcate the error message?

Reply:
What do you mean? I don't see that. Maybe you are confusing with the short() formatting on returnValue that after a certain length adds ...

Reply:
Ohh the single quotes? To contain the error message so it looks like `exceptionError='out of gas'`

Reply:
Yup, sorry, that comment wasn't clear but thanks for the clarification!